localisation/wc_culture_buildings.csv

```diff
@@ -1,212 +1,212 @@
+ca_culture_shathyar_1;Bastille of the Faceless;;Bastion der Gesichtslosen;;;;;;;;;;;x
+ca_culture_shathyar_2;Bastille of the Faceless;;Bastion der Gesichtslosen;;;;;;;;;;;x
+ca_culture_shathyar_3;Bastille of the Faceless;;Bastion der Gesichtslosen;;;;;;;;;;;x
+ca_culture_shathyar_4;Bastille of the Faceless;;Bastion der Gesichtslosen;;;;;;;;;;;x
+ca_culture_shathyar_desc;Seeping primordial forms of the Old Gods, the faceless are utterly fanatical in their devotion to their eldritch masters. They all had previously been sealed away by the titans, but now they have been broken free, enabling them to continue their devoted service once more.;;Die Gesichtslosen, die aus den Urformen der alten G�tter entsprungen sind, sind in ihrer Hingabe an ihre uralten Meister absolut fanatisch. Sie alle waren zuvor von den Titanen versiegelt worden, aber jetzt sind sie befreit worden, so dass sie ihren hingebungsvollen Dienst wieder aufnehmen k�nnen.;;;;;;;;;;;x
+ca_culture_sethrak_1;Krolusk Nesting Grounds;;Krolusk-Nistpl�tze;;;;;;;;;;;x
+ca_culture_sethrak_2;Krolusk Nesting Grounds;;Krolusk-Nistpl�tze;;;;;;;;;;;x
+ca_culture_sethrak_3;Krolusk Nesting Grounds;;Krolusk-Nistpl�tze;;;;;;;;;;;x
+ca_culture_sethrak_4;Krolusk Nesting Grounds;;Krolusk-Nistpl�tze;;;;;;;;;;;x
+ca_culture_sethrak_desc;The krolusks, seemingly native to wherever the sethrak inhabit, are a particularly resilient race of giant insects. The sethrak use these venomous bugs as companions, war mounts, and even as sources of food, making them almost absolutely essential to sethrak culture and way of life.;;Die Krolusken, die scheinbar �berall dort heimisch sind, wo die Sethrak leben, sind eine besonders widerstandsf�hige Rasse von Rieseninsekten. Die Sethrak benutzen diese giftigen K�fer als Gef�hrten, Kriegsreittiere und sogar als Nahrungsquelle, was sie f�r die Kultur und Lebensweise der Sethrak fast unentbehrlich macht.;;;;;;;;;;;x
+ca_culture_satyr_1;Hellcaster Enclave;;H�llenrufer-Enklave;;;;;;;;;;;x
+ca_culture_satyr_2;Hellcaster Enclave;;H�llenrufer-Enklave;;;;;;;;;;;x
+ca_culture_satyr_3;Hellcaster Enclave;;H�llenrufer-Enklave;;;;;;;;;;;x
+ca_culture_satyr_4;Hellcaster Enclave;;H�llenrufer-Enklave;;;;;;;;;;;x
+ca_culture_satyr_desc;Descended from the highborne elves of old, the satyr are magically extremely inclined as a people, though cursed by Xavius and his dark affliction. As a result of this, satyrs find themselves to be particularly talented in shadow and corrupting magic, regardless of who they claim allegience to.;;Von den Hochgeborenen der alten Elfen abstammend, sind die Satyrn als Volk magisch �u�erst begabt, obwohl sie von Xavius und seinem finsteren Gram verflucht sind. Dies hat zur Folge, dass Satyrn sich als besonders begabt in der Schatten- und korrumpierenden Magie erweisen, unabh�ngig davon, wem sie die Treue schw�ren.;;;;;;;;;;;x
```

Comment:
Moved extremely, fixed the spelling of allegiance. 
```suggestion
ca_culture_satyr_desc;Descended from the highborne elves of old, the satyr are extremely magically inclined as a people, though cursed by Xavius and his dark affliction. As a result of this, satyrs find themselves to be particularly talented in shadow and corrupting magic, regardless of who they claim allegiance to.;;Von den Hochgeborenen der alten Elfen abstammend, sind die Satyrn als Volk magisch �u�erst begabt, obwohl sie von Xavius und seinem finsteren Gram verflucht sind. Dies hat zur Folge, dass Satyrn sich als besonders begabt in der Schatten- und korrumpierenden Magie erweisen, unabh�ngig davon, wem sie die Treue schw�ren.;;;;;;;;;;;x
```

Reply:
thx